app/models/clinic.rb

```diff
@@ -17,13 +17,33 @@ class Clinic
   field :active, type: Boolean, default: true
   field :accepts_naf, type: Boolean, default: false
   field :gestational_limit, type: Integer
-  field :costs_9wks, type: Integer
+  field :costs_05wks, type: Integer
+  field :costs_06wks, type: Integer
+  field :costs_07wks, type: Integer
```

Comment:
I wonder if we can do this with a range of some sort -- I THINK, though I haven't tried it, that mongoid will let you dynamically define model fields, since it's all ruby code anyway. Maybe something like `(5..30).each { |i| field "costs_#{i}wks".to_sym, type: Integer }`?